webrender/src/device.rs

```diff
@@ -863,9 +859,13 @@ impl Device {
 
             next_vao_id: 1,
             //file_watcher: file_watcher,
+
+            max_texture_size: gl::get_integer_v(gl::MAX_TEXTURE_SIZE) as u32
         }
     }
 
+    pub fn max_texture_size(&self) -> u32 { self.max_texture_size }
```

Comment:
nit: There should be a new line here (unless there is something in the rust style guide specifically about this?).